tests/unit/specs/__snapshots__/TopBar.spec.js.snap

```diff
@@ -0,0 +1,37 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
do we need this file? It seems to be regenerated on every test run

Reply:
Yes, this file is used for comparison with the generated one. It will fail the test in case the HTML of the component would change. If the change is intentional, tests need to be run with `yarn test:unit -u` and then commit and push the updated file.